python/gui/auto_generated/editorwidgets/core/qgswidgetwrapper.sip.in

```diff
@@ -165,6 +165,15 @@ that any pending changes should be pushed to the edit buffer or they
 might be lost.
 
 .. versionadded:: 3.2
+%End
+
+  signals:
+
+    void contextChanged();
+%Docstring
+Signal when QgsAttributeEditorContext mContext changed after the widget wrapper has already been intialized
```

Comment:
Not sure if I understand your question. Ìt can be connected on creation of widget wrappers, in case they need to make an action. Like `QgsQmlWidgetWrapper`...

Reply:
I was mostly wondering why the part "after the widget wrapper has already been initialized" is in the doc string.

Reply:
So it's mostly a historical consideration?
In this case I don't think it's useful for future readers of this docs.

Reply:
No not historical. E.g. the `QgsQmlWidgetWrapper` used the context to set values. On change of the context, these values has to be changed as well. That's why the signal is implemented, to trigger these changes, when the context changes *after* the Widget is initialized. If it changes before, then the inexistent Widged doesn't mind anyway. But probably it's too clear, so it only confuses...